### PR TITLE
[JENKINS-55994] Automated test more specific for java 11

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-55994](https://issues.jenkins-ci.org/browse/JENKINS-55994)

Automated test for JDK:
- Using JDK 11 features.
- Specifying cpu and memory options in the container.

### CODE REVIEW

**Comment by varyvol** [comment]
@oleg-nenashev please be aware that jaxb is not installed because install-plugins.sh does not handle detached plugins.

File: tests/test_resources/cwp-produced-images/config/packager-config-jdk11-more-specific.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        dir: ./.jenkinsfile-runner-test-framework/source
+    docker:
+      base: "jenkins/jenkins:2.164-jdk11"
+      tag: "jenkins-experimental/jenkinsfile-runner-demo"
+      build: false
+war:
+  groupId: "org.jenkins-ci.main"
+  artifactId: "jenkins-war"
+  source:
+    version: "2.164"
```
Comment (fcojfernandez) [inline tests/test_resources/cwp-produced-images/config/packager-config-jdk11-more-specific.yml:24]: Why not 2.164.1? I think it's the first LTS java-11 compatible.

**Review by fcojfernandez — APPROVED** [review]

File: tests/test_resources/cwp-produced-images/config/packager-config-jdk11-more-specific.yml (hunk 1 above)
Comment (varyvol) [reply tests/test_resources/cwp-produced-images/config/packager-config-jdk11-more-specific.yml:24]: It's the same one that in the previous jdk 11 configuration we were using.

Comment (fcojfernandez) [reply tests/test_resources/cwp-produced-images/config/packager-config-jdk11-more-specific.yml:24]: But 2.164.1 wasn't released at that moment. Maybe is it worth updating?

**Review by oleg-nenashev — APPROVED** [review]
I would prefer to see a user-facing test like SLOCCount plugin as it is written in the https://issues.jenkins-ci.org/browse/JENKINS-55994 acceptance criteria. But I am fine with that test, it definitely adds the Java 11 spotcheck

**Comment by varyvol** [comment]
@oleg-nenashev I did not include the SLOCCount thing because jaxb does not get automatically installed.

**Pull request merged**